vault/resource_consul_secret_backend.go

```diff
@@ -3,12 +3,13 @@ package vault
 import (
 	"context"
 	"fmt"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
-	"github.com/hashicorp/terraform-provider-vault/internal/consts"
-	"github.com/hashicorp/terraform-provider-vault/internal/semver"
 	"log"
 	"strings"
 
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
```

Comment:
Formatting seems a bit off, local package imports should be grouped together.

Reply:
This happened due to Goland's formatting for sorting imports, not sure what the right order is in your files but Goland reformats these for me upon saves

Reply:
Fixed